### PR TITLE
add members at creation time

### PR DESCRIPTION
**Pull request by b97pla**
Additional members that should be added to the delivery project at creation time can be specified on the command line.

### CODE REVIEW

File: deliver.py [hunk 1]
```diff
@@ -91,6 +91,9 @@ def create_delivery_project(base_url, project_name, pi_id, sensitive_data, user,
 parser.add_argument("-s", "--staging_area", help="Path to the directory where directory should"
                                                  " be staged prior to delivery", required=True)
 parser.add_argument("-e", "--email", help="Email address to the PI (Must be same as in Supr)", required=True)
+parser.add_argument("-m", "--member_email", required=False, nargs="*",
+                    help="Email address to additional memebers that should be added to the delivery project in "
```
Comment (johandahlberg) [inline deliver.py]: `memebers` -> `members `

File: deliver.py [hunk 2]
```diff
@@ -144,12 +148,19 @@ def create_delivery_project(base_url, project_name, pi_id, sensitive_data, user,
...
-                              key=supr_api_key)
+pi_member_id = map(
+    lambda email: search_pi_id_by_email(
+        base_url=supr_base_url,
+        email=email,
+        user=supr_api_user,
+        key=supr_api_key),
+    [pi_email] + member_email)
+pi_id = pi_member_id[0]
+member_id = pi_member_id[1:]
```
Comment (johandahlberg) [inline deliver.py]: Perhaps this variable could be called `member_ids` to make it clearer that it can contain more than one member id?

**Review by johandahlberg — APPROVED** [review]
Some minor comments. But this looks good to me.

**Comment by b97pla** [comment]
thanks, I've updated the PR. Should be good to merge now.

**Pull request merged**